src/puppetlabs/trapperkeeper/authorization/ring_middleware.clj

```diff
@@ -202,8 +202,8 @@
 
 (schema/defn add-authinfo :- ring/Request
   "Add authentication information to the ring request."
-  [request :- ring/Request
-   allow-header-cert-info :- schema/Bool]
+  [allow-header-cert-info :- schema/Bool
+   request :- ring/Request]
```

Comment:
I swapped these parameters just for the convenience of being able to thread the request in a `->>` pipeline in `authorization-check`